src/mlpack/tests/main_tests/adaboost_fit_test.cpp

```diff
@@ -0,0 +1,319 @@
+/**
+ * @file tests/main_tests/adaboost_fit_test.cpp
```

Comment:
I think if we rename this `adaboost_train_test` and adapt the includes/other bits to use `train` instead of `fit` (and do the same for the linear regression test), then the build will pass. :+1: